osidb/models.py

```diff
@@ -2485,6 +2486,29 @@ def _validate_tracker_ps_update_stream(self, **kwargs):
                 "Tracker must be associated with a valid PS update stream"
             )
 
+    def _validate_tracker_ps_module_ps_update_stream(self, **kwargs):
+        """
+        check that the tracker is associated with a PS update stream corresponding to its PS module
+        """
+        if not self.affects.exists():
+            return
+
+        ps_module = PsModule.objects.filter(name=self.affects.first().ps_module)
```

Comment:
I believe you need to add `first()` or this will fail at 2506:
```suggestion
        ps_module = PsModule.objects.filter(name=self.affects.first().ps_module).first()
```

Reply:
You are correct, thank you!